docker/project-fortis-services/run-server.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+cassandra_exec() {
+  /opt/cassandra/bin/cqlsh \
+    --request-timeout=3600 \
+    --username="$CASSANDRA_USERNAME" \
+    --password="$CASSANDRA_PASSWORD" \
+    "$CASSANDRA_CONTACT_POINTS"
+}
+
+# wait for cassandra to start
+while ! cassandra_exec; do
+  echo "Cassandra not yet available, waiting..."
+  sleep 10s
+done
+echo "...done, Cassandra is now available"
+
+# set up cassandra schema
+if [ -n "$FORTIS_CASSANDRA_SCHEMA_URL" ]; then
+  echo "Got Fortis schema definition at $FORTIS_CASSANDRA_SCHEMA_URL, ingesting..."
+  wget -qO- "$FORTIS_CASSANDRA_SCHEMA_URL" | cassandra_exec
```

Comment:
Excellent! At some point, I think we could move these DDL statements into database migrations running on this project (fortis-services).